### PR TITLE
Last contributions from eosc-synergy/FAIR_eva

### PR DESCRIPTION
**Pull request by orviz**
Including:

- Plugin for EPOS
- Improved lookup of config.ini files (regardless of the path where the `fair.py` is invoked)
- Source code format using [black](https://pypi.org/project/black/), enforced through [pre-commit](https://pre-commit.com/)
- Improved docstrings
- Implementation of GitHub workflows for managing back-and-forth syncs between synergy and IFCA-Advanced-Computing orgs

### CODE REVIEW

File: petition.py
Comment (ferag) [inline petition.py:1]: Is this for testing? Do you think root path is the best to put this function?

File: plugins/digital_csic/plugin.py [hunk 1]
```diff
@@ -13,80 +12,73 @@
...
+)
 
-class Plugin(Evaluator):
+logger = logging.getLogger(os.path.basename(__file__))
 
-    """
-    A class used to represent an Animal
 
-    ...
+class Digital_CSIC(Evaluator):
```
Comment (ferag) [inline plugins/digital_csic/plugin.py:22]: Class need to be called Plugin(Evaluator)

File: plugins/example_plugin/plugin.py [hunk 2]
```diff
@@ -8,101 +8,109 @@
...
 
 
-
-class Plugin(Evaluator):
-
-    """
-    A class used to represent an Animal
-
-    ...
+class Example_Plugin(Evaluator):
```
Comment (ferag) [inline plugins/example_plugin/plugin.py:17]: Class need to be called Plugin(Evaluator)

**Review by ferag — COMMENTED** [review]
Take a look on my comments ;) thanks!

File: petition.py
Comment (orviz) [reply petition.py:1]: moved to a `scripts` folder?

File: plugins/digital_csic/plugin.py (hunk 1 above)
Comment (orviz) [reply plugins/digital_csic/plugin.py]: done

File: plugins/example_plugin/plugin.py (hunk 2 above)
Comment (orviz) [reply plugins/example_plugin/plugin.py]: done

**Review by ferag — APPROVED** [review]

**Pull request merged**